tools/release-management/src/bin/version_updater.rs

```diff
@@ -0,0 +1,208 @@
+    repo.checkout_new_branch(&branch_name)
+        .expect("Failed to create branch");
+    tracing::info!(%branch_name, ?changes);
+
+    repo.add(&[
+        go_constants_path.to_str().unwrap(),
+        cargo_toml_path.to_str().unwrap(),
+    ])
+    .expect("Failed to add files");
+    repo.commit(COMMIT_MESSAGE).expect("Failed to commit");
+    repo.push(&branch_name).expect("Failed to push");
+
+    // Create PR
+    let authed_github = octocrab::OctocrabBuilder::default()
+        .personal_token(github_token)
+        .build()?;
+
+    // tracing::info!(%owner, %repo_name, ?branch_name, ?pr_message);
+    authed_github
+        .pulls(owner, repo_name)
+        .create(PR_TITLE, branch_name, "master")
+        .body(COMMIT_MESSAGE)
+        .send()
+        .await?;
+
+    Ok(())
+}
+
+async fn set_rust_sdk_version(args: &Args, cargo_toml_path: &PathBuf) -> eyre::Result<()> {
+    let cargo_toml_string = std::fs::read_to_string(cargo_toml_path)?;
+    let mut cargo_toml = cargo_toml_string.parse::<DocumentMut>()?;
+    cargo_toml["workspace"]["package"]["version"] = toml_edit::value(args.sdk_version.clone());
+    std::fs::write(cargo_toml_path, cargo_toml.to_string())?;
+    Ok(())
+}
+
+async fn set_go_sdk_version(args: &Args, go_constants_path: &PathBuf) -> eyre::Result<()> {
+    let go_constants_source_code_string = std::fs::read_to_string(go_constants_path)?;
+    let updated_go_source_code = go_constants_source_code_string.replace(
+        go_constants_source_code_string
+            .lines()
+            .find(|line| line.contains("sdkVersion ="))
+            .unwrap(),
+        &format!("const sdkVersion = \"{}\";", &args.sdk_version),
+    );
+
+    std::fs::write(go_constants_path, updated_go_source_code)?;
+    Ok(())
+}
+
+#[cfg(test)]
+mod test {
+
+    #[tokio::test]
+    async fn test_set_rust_sdk_version() {
+        let args = super::Args {
+            root: String::from("."),
+            git_dir: String::from("."),
+            github_repo: String::from("."),
+            sdk_version: String::from("0.2.1"),
+            rust_root: String::from("rust"),
+            go_root: String::from("go"),
+        };
+        let cargo_toml_path = std::path::PathBuf::from("test_resources").join("Cargo.toml");
+
+        super::set_rust_sdk_version(&args, &cargo_toml_path)
+            .await
+            .unwrap();
+
+        let cargo_toml_string = std::fs::read_to_string(&cargo_toml_path).unwrap();
+        let cargo_toml = cargo_toml_string.parse::<toml_edit::DocumentMut>().unwrap();
+
+        let version = cargo_toml
+            .get_key_value("workspace")
+            .unwrap()
+            .1
+            .get("package")
+            .unwrap()
+            .get("version")
+            .unwrap()
+            .to_string()
+            .replace(['\\', '\"'], "")
+            .trim()
+            .to_string();
+
+        assert!(version == "0.2.1");
```

Comment:
if you have an inline string you can then check for file equality as a whole

Reply:
Done